core/server/common/src/main/java/alluxio/ProcessUtils.java

```diff
@@ -37,11 +37,13 @@ public static void run(Process process) {
       System.exit(0);
     } catch (Throwable t) {
       LOG.error("Uncaught exception while running {}, stopping it and exiting.", process, t);
+      LOG.error("Root cause of the exception: ", ExceptionUtils.getRootCause(t));
```

Comment:
Can we include this in the previous log message?

Reply:
done